mongo.js

```diff
@@ -241,13 +247,29 @@ module.exports.connect = function( url, options ) {
     return conn;
 }
 
+module.exports.destroy = function() {
+    for ( const url in connections ) {
+        debug('closing', url);
+
+        clearTimeout( connections[ url ].closeTimeout )
+        delete connections[ url ].closeTimeout;
+
+        if( connections[ url ].client ) {
```

Comment:
What happen if there is more than 1 client on the same `connections[ url ]`?
https://github.com/panoplyio/dbstream-mongo/blob/58e1be343aff933877414cd2592b2be4c7917362/mongo.js#L142

Reply:
As far as I can tell there's only 1 actual client per `connections[ url ]` so closing it should do the trick. When a client is added all that is affected is a counter.